### PR TITLE
Add NASA certification process

### PR DESCRIPTION
**Pull request by iramosp**
This PR adds a file explaining the NASA certification process to the Nebula cohort processes and procedures folder. 

- Tagging @doaamkader for content review: Are the steps clear? Any questions we should anticipate?
- Tagging @NPDebs for technical review after content approval. 

:+1::tada: First of all, thanks for taking the time to contribute! :tada::+1:

## FOR CONTRIBUTOR

* [ ] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [ ] This PR adds a new blog post
* [x] This PR does something else (explain above)

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [x] The website builds (a check will fail if not)
* [ ] All images have been added within the Pull Request and they have Alt text
* [ ] If there are paragraphs or text, the key messages are highlighted
* [ ] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [ ] The preview corresponds to the changes described in the Pull Request
* [ ] The code is tidy and passes the linting tests

### CODE REVIEW

**Comment by NPDebs** [comment]
Thanks, Irene! 🌟 

@doaamkader - Here's [a preview](https://deploy-preview-926--ols-bebatut.netlify.app/nebula/nebula-cohort-procedures/nasa-certification.html) of the changes made.

File: nebula/nebula-cohort-procedures/nasa-certification.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  url: https://images.unsplash.com/photo-1526212076326-45d0c30a89e8
+---
+
+**Congratulations!** Upon graduating from [Nebula](https://we-are-ols.org/nebula/) or [Open Seeds](https://we-are-ols.org/openseeds/), you will earn a NASA TOPS Badge via [Credly](https://www.credly.com/organizations/nasa-tops/badges) certifying your knowledge of Open Science best practices. Please follow these steps to finalize the certification process. 
+
+## Steps to claim your NASA certificate
+
+1. **Create an [ORCID](https://orcid.org/) account** if you don't have one already. 
+
+2. **Create and log into an account on https://openscience101.org/.***
```
Comment (NPDebs) [inline nebula/nebula-cohort-procedures/nasa-certification.md:17]:
```suggestion
2. **Create and log into an account on [openscience101.org](https://openscience101.org/).** 

```

File: nebula/nebula-cohort-procedures/nasa-certification.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+3. **Complete the post-cohort feedback survey.**
+- The OLS team will provide a link to the feedback survey after the cohort ends. 
+- You will need to submit the following information for your certificate: full name, ORCID, and the email associated with your ORCID. 
+
+4. **Accept your NASA badge.**
+- Within 30 days of submitting your information, you will receive an email from Credly inviting you to accept your digital badge. 
+- Once accepted, the badge will appear in your ORCID account.
+
+### Contact
+For any questions about this process, please contact us at nebula-cohort-management@we-are-ols.org.
```
Comment (NPDebs) [inline nebula/nebula-cohort-procedures/nasa-certification.md:31]:
```suggestion
For any questions about this process, please contact us at [{{ site.nebula_email }}@{{ site.email_extension }}](mailto:{{ site.nebula_email }}@{{ site.email_extension }}).

```

**Review by NPDebs — APPROVED** [review]
From my side, this looks good @iramosp! 🎉  Happy to merge after content review from Doaa.

I have added a couple of tweaks in your PR which I should let you know about:
- Made openscience101.org link clickable.
- Made Nebula resources findable by adding them to the "Resources" dropdown menu in the navbar.
- Created a new variable (site.`nebula_email`) to store the Nebula contact email (nebula-cohort-management@...).
- Used the variable to replace the explicit mention of the email address.

Let me know if any of these don't sit right with you, or need more clarification. Thanks again!

**Comment by iramosp** [comment]
@doaamkader, thanks for checking!
@NPDebs, thanks for going the extra mile to add those changes! 

I think this is ready to merge 🙌

**Comment by NPDebs** [comment]
Thank you both for the work here! 🥳💪🏽

**Pull request merged**